new/LIT7072QeneMamhera.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7072QeneMamhera" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mamǝhǝra zǝntu ʿālam, ʾAwǝsṭātewos, dabra ʾArmānyā waldaka baṣḥa la-zammǝro… (Qǝne of unknown type)</title>
+                <title xml:lang="en" corresp="#t1">The teacher of this world, ʾEwosṭātewos, came to the mountain of Armenia to praise Your Son…</title>
```

Comment:
I wonder, if this can also be translated as "came to **a** mountain **in** Armenia" or "came to **a monastery in** Armenia".

Reply:
It is possibly the monastery called ደብር፡ ለስርአተ፡ ሌዊ፡, which is mentioned in the text in line 76.

Reply:
Well, you can never know if dabr is "monastery" or "mountain" if it's not clear from the context, right? I think the mention of "dabr" later in the qene is not a name.

Reply:
OK.